DrewBradley/rancid-tomatillos
# Feature/network requests

### What does this PR do?
- This PR shifts our data from local to API to be used to display all movies and an individual movie with additional details.

### What does it fix?
- No bugs to fix.

### Is this a feature or a fix?
- This is a feature, pretty much completing iteration 3 aside from some extra styling of components.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 

### How should this be tested?
- Check out the page once it is loaded. Click on different movies to verify the page changes to an individual movie view with a large screenshot in the background, as well as a small poster thumbnail, and details like runtime, revenue, budget, rating, etc.
